ui/component/livestreamComments/view.jsx

```diff
@@ -286,7 +343,13 @@ export default function LivestreamComments(props: Props) {
 
           {pinnedComment && showPinned && viewMode === VIEW_MODES.CHAT && (
             <div className="livestream-pinned__wrapper">
-              <LivestreamComment comment={pinnedComment} key={pinnedComment.comment_id} uri={uri} />
+              <LivestreamComment
+                comment={pinnedComment}
+                key={pinnedComment.comment_id}
+                uri={uri}
+                dateLastUpdate={dateLastUpdate}
```

Comment:
With the other suggested changes above, maybe renaming `dateLastUpdate` to `forceUpdate` might be more appropriate?  Just to prevent other people from confusing this as a "last render variable", which it is not.